### PR TITLE
Fill GIV price for donations on mainnet

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
Now https://givback.develop.giveth.io/api-docs/#/default/get_givPrice support mainnet price of GIV, so I made some changes on filling GIV pries

### CODE REVIEW

File: src/entities/donation.ts [hunk 1]
```diff
@@ -128,12 +128,8 @@ export class Donation extends BaseEntity {
 
   // these are all the donations before monoswap was updated
```
Comment (CarlosQ96) [inline src/entities/donation.ts]: Remove this comment, this was when I first did the script. So we don't get confused later.

Comment (CarlosQ96) [reply src/entities/donation.ts]: Or update it to something more related to the current code.

**Review by CarlosQ96 — APPROVED** [review]
Everything else looks good! Great job man!

**Comment by CarlosQ96** [comment]
@mohammadranjbarz actually now that I noticed we are not updating the projects total amount donated. So should we make a separate issue or update this script? I can add it this weekend.

Ashely Giv donations did not affect the total amount. in this issue: https://github.com/Giveth/giveth-next/issues/643

File: src/entities/donation.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply src/entities/donation.ts]: Thanks , I deleted the outdated comment

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz actually now that I noticed we are not updating the projects total amount donated. So should we make a separate issue or update this script? I can add it this weekend.
> 
> Ashely Giv donations did not affect the total amount. in this issue: [Giveth/giveth-next#643](https://github.com/Giveth/giveth-next/issues/643)

Good point, Now I call  
```
await updateTotalDonationsOfProject(donation.projectId)
```
 After updating `valueUsd` of donations.
For projects with invalid totalAmount we can wait till some donate to them, after `totalRaisedAmount` would fixed

**Comment by CarlosQ96** [comment]
Perfect Thanks man!

**Review by CarlosQ96 — APPROVED** [review]

**Pull request merged**